### PR TITLE
[Localization] Add localization for able/not able/learned

### PR DESCRIPTION
**Pull request by hayuna**
## What are the changes?
Localization for TM learning

## Why am I doing these changes?
Hardcoded text -> localized text

## What did change?
Added localization (from Google Translate)

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/party-ui-handler.ts [hunk 1]
```diff
@@ -7,5 +7,8 @@ export const partyUiHandler: SimpleTranslationEntries = {
   "CANCEL": "Annuler",
   "RELEASE": "Relâcher",
   "APPLY": "Appliquer",
-  "TEACH": "Apprendre"
+  "TEACH": "Apprendre",
+  "ABLE": "Capable",
+  "NOT_ABLE": "Pas capable",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:12]:
```suggestion
  "ABLE": "Apte",
  "NOT_ABLE": "Non apte",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
Translation mistakes

**Comment by Dakurei** [comment]
Please never use online translators again in the future, but leave the default English text for native speakers to handle

**Comment by InnocentGameDev** [comment]
Please do not add the Spanish yourself, always check with translators and we'll take care of it (from the English default).

**Comment by CodeTappert** [comment]
Never add localized text to languages you dont speak. 
Use english text al placeholder instead

**Comment by CodeTappert** [comment]
Also i will close this as this text is already in a PR. See https://github.com/pagefaultgames/pokerogue/pull/2566/files

So this is a duplicate

**Comment by hayuna** [comment]
got it

**Pull request closed without merging**